sysutils/dcfldd/Portfile

```diff
@@ -1,29 +1,31 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                dcfldd
-version             1.3.4-1
-revision            1
-checksums           rmd160  3a6fba197686da8f1890d21f75b6bc2ae18592a7 \
-                    sha256  f5143a184da56fd5ac729d6d8cbcf9f5da8e1cf4604aa9fb97c59553b7e6d5f8 \
-                    size    163141
+github.setup        ressurecting-open-source-projects dcfldd 1.9.1 v
+github.tarball_from releases
 
+name                dcfldd
+revision            0
 categories          sysutils
-maintainers         nomaintainer
+maintainers         {@davidpolverari gmail.com:davidpolverari} \
+                    openmaintainer
+description         enhanced version of dd for forensics and security
+long_description    ${name} is an enhanced version of the dd command with \
+                    additional features useful for forensics and information \
+                    security, designed to run on Unix or Unix-like \
+                    (POSIX-compliant) operating systems.
+
+homepage            https://github.com/resurrecting-open-source-projects/dcfldd
 license             GPL-2+
 
-description         Enhanced dd with md5 checksums
-long_description    Enhanced dd with md5 checksums
-
-homepage            http://dcfldd.sourceforge.net/
-platforms           darwin
-
-master_sites        sourceforge
-
-configure.args  --mandir=${prefix}/share/man
+fetch.type          git
```

Comment:
no, this should not be set. Why do you think this is needed?

Reply:
It didn't work locally without this statement.

Reply:
I still couldn't make it work without `fetch.type git`. If I use only `fetch.type git` without `git.url`, a prompt appears asking for GitHub username. It only works for me with both `fetch.type` and `git.url`. Otherwise, MacPorts can't fetch dcfldd.

Reply:
both statements shouldn't be needed and thus removed. Did you verify upstream generates "releases" - if that is not the case, then the "github.tarball_from" line should be changed, likely to "archive". If that's the case you'll likely need to update the checksums as well.

Reply:
Yes, upstream generates releases, as it can be seen [here](https://github.com/resurrecting-open-source-projects/dcfldd/releases). I can send the changes without those two lines, but I haven't been able to install it locally without them. I don't know what else might be the problem atm.

Reply:
Let me take a look in a bit when I'm behind a computer. I **think** it should work witjout these lines from a cursory read of the Portfile.